src/po/page/board.page.js

```diff
@@ -2,14 +2,82 @@ import {
   BoardHeader,
   BoardCanvas,
   HeaderComponent,
-  GenerateRandomTitles,
+  RandomTitleGenerator,
 } from "../components/index.js";
 
+import { assert, expect, should } from "chai";
+should();
+
 export class BoardPage {
   constructor() {
     this.headerComponent = new HeaderComponent();
     this.boardHeader = new BoardHeader();
     this.boardCanvas = new BoardCanvas();
-    this.generateRandomTitles = new GenerateRandomTitles();
   }
+
+  //User searches for a board
+async userSearchForBoard () {
+  await this.headerComponent.item("search").waitForDisplayed();
+  await this.headerComponent.item("search").click();
+  await this.headerComponent.item("searchInput").setValue("Example test case");
+  await this.headerComponent.item("searchResult").click();
+  const title = await browser.getTitle();
+  assert.equal(title, "Example test case | Trello", `The board hasn't been found`);
+}
+
+//User creates new lists on a board
+async userCreatesNewList () {
+  const listTitle = RandomTitleGenerator.titleGenerator("list");
+  await this.boardCanvas.item("addList").click();
+  await this.boardCanvas.item("listValue").setValue(listTitle);
+  await this.boardCanvas.item("addListBtn").click();
+  const listTitleText = await this.boardCanvas.getListByTitle(listTitle).getText();
+  expect(listTitleText).to.equal(listTitle);
+  const listTitleDisplay = await this.boardCanvas.getListByTitle(listTitle).isDisplayed();
+  expect(listTitleDisplay, `List with title - "${listTitle}" isn't displayed`).to.be.true;
+}
+
+//User creates a new card in a list
+async userCreatesNewCard () {
+  const cardName = RandomTitleGenerator.titleGenerator("card");
+  await this.boardCanvas.addNewCard().click();
+  await this.boardCanvas.item("cardValue").setValue(cardName);
+  await this.boardCanvas.item("addCardBtn").click();
+  const cardNameText = await this.boardCanvas.getCardByName(cardName).getText();
+  expect(cardNameText).to.equal(cardName);
+  const cardNameDisplay = await this.boardCanvas.getCardByName(cardName).isDisplayed();
+  expect(cardNameDisplay, `Card with name - "${cardName}" isn't displayed`).to.be.true;
+}
+
+//User filtering of cards by label
+async userFilteringCardByLabel () {
```

Comment:
if you decided to use this method only for urgent filtering add it to func name, filterCardByUrgentLabel, as we have test for other filterings how will you call them?